Source/WebKit/DerivedSources-input.xcfilelist

```diff
@@ -375,9 +375,12 @@ $(PROJECT_DIR)/WebProcess/Extensions/Interfaces/WebExtensionAPIExtension.idl
 $(PROJECT_DIR)/WebProcess/Extensions/Interfaces/WebExtensionAPINamespace.idl
 $(PROJECT_DIR)/WebProcess/Extensions/Interfaces/WebExtensionAPIPermissions.idl
 $(PROJECT_DIR)/WebProcess/Extensions/Interfaces/WebExtensionAPIRuntime.idl
+$(PROJECT_DIR)/WebProcess/Extensions/Interfaces/WebExtensionAPITabs.idl
```

Comment:
Will there be a TabsEvent.idl as well?

Reply:
No, only Windows has a special event since it has a filter. Tabs uses the generic event already in the code.